### ISSUE TITLE
Double inheritance warning not emitted

### ISSUE THREAD

**Issue by socketpair** [PyCQA/pylint#6991]
### Bug description

asdfasdfasdf

### Command used

```shell
asdfasdf
```

### Pylint output

```shell
1asdfasdfasdf
```

### Expected behavior

sadfasdf

### Pylint version

```shell
1
```
